### DISCUSSION TITLE
IzzyOnDroid claims 1.1.1 is the newest version

### DISCUSSION THREAD

**Discussion by v1s7** [samolego/Canta#43] [General]
The weird thing is you can update to IOD's 1.1.1 without any problems, no matter what version you have installed. But if you try to return to any F-Droid version the installer will claim it's older than the existing one. What's the reason behind this?
![vQ6uvpnpz1BSPpGQp50nKmsf0HgzNjV0](https://github.com/samolego/Canta/assets/86906143/a266a9c8-a011-4beb-bfb0-170a73935b13)

**Comment by samolego** [comment]
Seems like I might have messed version number in 1.1.1 :/ ...

**Reply by v1s7** [reply]
Should I make this into an issue?

**Reply by samolego** [reply]
Idk. I guess I could remove previous releases <= 1.1.1 ...

**Reply by samolego** [reply]
Though I don't like the sollution

**Reply by v1s7** [reply]
Can't you just remove the app from IOD completely? The F-Droid's 1.1.1 works as intended.

**Reply by samolego** [reply]
No, I've not added it to IOD even. I've only added it to fdroid.
